source/elements/oneMKL/source/domains/rng/device_routines/device-distributions.rst

```diff
@@ -0,0 +1,95 @@
+.. SPDX-FileCopyrightText: 2023 Intel Corporation
+..
+.. SPDX-License-Identifier: CC-BY-4.0
+
+.. _onemkl_device_rng_distributions:
+
+Device Distributions
+====================
+
+
+oneMKL RNG routines are used to generate random numbers with different types of distribution. Each function group is 
+introduced below by the type of underlying distribution and contains a short description of its functionality, as well 
+as specifications of the call sequence and the explanation of input and output parameters. The Device Continuous 
+Distribution Generators table and Device Discrete Distribution Generators table list the random number generator routines 
+with data types and output distributions, and sets correspondence between data types of the generator routines and the 
+basic random number generators.
+
+**Device Continuous Distribution Generators**
+
+.. list-table::
+   :header-rows: 1
+
+   * -     Type of Distribution
+     -     Data Types
+     -     BRNG Data Type
+     -     Description
+   * -     :ref:`onemkl_device_rng_uniform_continuous`
+     -     s, d
```

Comment:
What do you mean by `s, d`? Is it `float, double`? Let's say it here explicitly here instead of `s, d`

Reply:
It is aligned with out Developer Reference. Do you suggest to change it everywhere? If so, I'm not sure how to handle parts with `i` as it may be std::{u}int{32/64}_t

Reply:
Spec shouldn't be aligned with the Developer Reference. Spec is not directly related to the product. The oneMKL product implements the spec, but Spec can be implemented by different vendors, not only Intel

Reply:
moved to float and double